File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/command/TimeStampGetCmd.kt
package com.syncrhrohealth.bandpass_ble_connection.core.command

import com.syncrhrohealth.bandpass_ble_connection.core.command.builder.CommandBuilder
import com.syncrhrohealth.bandpass_ble_connection.core.enum.CommandCode
import com.syncrhrohealth.bandpass_ble_connection.core.handler.DeviceHandler


object TimeStampGetCmd {
    fun send(handler: DeviceHandler) {
        val builder = CommandBuilder.build(CommandCode.PACKET_CMD_TIMESTAMP_GET)
        handler.write(builder)
    }
}

File: ios/Classes/Core/Command/TimeStampGetCmd.swift
//
//  TimeStampGetCmd.swift
//  Runner
//
//  Created by MAC on 11/3/25.
//

import Foundation

struct TimeStampGetCmd {
    static func send(handler: DeviceHandler) {
        let builder = CommandBuilder.build(commandCode: .packetCmdTimestampGet)
        handler.write(data: builder)
    }
}
